Jenkinsfile

```diff
@@ -805,15 +805,15 @@ def archiveTestOutput(Map args = [:]) {
           }
         }
         def fileName = 'build/system-tests-*.tar.gz' // see dev-tools/mage/target/common/package.go#PackageSystemTests method
-        dir("${BASE_DIR}"){
-          def files = findFiles(glob: "${fileName}")
-          files.each { file ->
-            echo "${file.name}"
-          }
+        def files = findFiles(glob: "${fileName}")
+        files.each { file ->
+          echo "${file.name}"
+        }
+        dir("build"){
```

Comment:
IIUC,  `dir("build"){` is not required since the L816 uses the `fileName` that contains the build folder

Reply:
Ah, that's right! Good catch!